File: app/src/main/java/dag/lydbok/exception/LydbokException.java
package dag.lydbok.exception;

public class LydbokException extends Exception {
    public LydbokException(String message, Throwable cause) {
        super(message, cause);
    }

    @Override
    public String toString() {
        return "LydbokException{}" + getMessage() + "/" + getCause().toString() ;
    }
}

File: app/src/main/java/dag/lydbok/model/LydbokException.kt
package dag.lydbok.model

class LydbokException : Exception {
    constructor(message: String, cause: Throwable) : super(message, cause) {}

    constructor(message: String) : super(message) {}

    override fun toString(): String {
        return "LydbokException{}$message/${cause.toString()}"
    }
}
